docs/src/contribute/pull-requests.md

```diff
@@ -145,27 +141,24 @@ Now you're ready to send the pull request. Go to your ESLint fork and then follo
 
 In order to submit code or documentation to an ESLint project, you’ll be asked to sign our CLA when you send your first pull request. (Read more about the Open JS Foundation CLA process at <https://cla.openjsf.org/>.)
 
+The pull request title is autogenerated from the summary of the first commit, but it can be edited before the pull request is submitted.
+
+The description of a pull request should explain what you did and how its effects can be seen.
+
+When a pull request is merged, its commits will be squashed into one single commit. The first line of the squashed commit message will contain the title of the pull request and the associated issue number.
```

Comment:
```suggestion
When a pull request is merged, its commits will be squashed into one single commit. The first line of the squashed commit message will contain the title of the pull request and the pull request number.
```

So that it is not confused with an issue the PR addresses (if the PR addresses an issue).